bin/martinize2

```diff
@@ -726,6 +728,17 @@ def entry():
             message = 'Unknown value for -eunit: "{}".'.format(args.rb_unit)
             LOGGER.critical(message)
             raise ValueError(message)
+        if args.rb_region == 'molecule':
+            domain_criterion = vermouth.processors.apply_rubber_band.always_true
+            regions = []
+        else:
+            regions = [tuple([int(i) for i in apair.split(':')]) for apair in args.rb_region.split(',')]
+            if len([i for i in [len(atuple) for atuple in regions] if i != 2])==0:
```

Comment:
nit: these lines are rather complex, and might benefit from being split into smaller bits and/or comments